instrumentation/build.gradle.kts

```diff
@@ -79,8 +75,6 @@ dependencies {
     testImplementation(libs.androidx.test.core)
     testImplementation(libs.assertj.core)
     testImplementation(libs.awaitility)
-
-    coreLibraryDesugaring(libs.desugarJdkLibs)
```

Comment:
Why remove this at the top level?

Reply:
It was moved over to `otel.android-library-conventions.gradle.kts` to ensure that all android libs created in this repo are corelib-aware.